### ISSUE TITLE
Request for a new release of ESLint.Formatter

### ISSUE THREAD

**Issue by Yiwei-Ding** [microsoft/sarif-sdk#2452]
Since #2435 had been merged in, could we have a new release of ESLint.Formatter so that we can leverage the new feature?

**Comment by eddynaka** [comment]
Hi @Yiwei-Ding , we are working on some improvements for the formatter as well:
https://github.com/microsoft/sarif-sdk/pull/2444

Once we merge that, we will probably release it.

@EasyRhinoMSFT @yongyan-gh

**Comment by Yiwei-Ding** [comment]
Still waiting for #2458

**Comment by yongyan-gh** [comment]
#2458 was merged today

**Comment by eddynaka** [comment]
Hi @Yiwei-Ding ,

we just released:
https://www.npmjs.com/package/@microsoft/eslint-formatter-sarif

Let us know if you face any problems.
Thank you!

**Comment by yongyan-gh** [comment]
I got below errors in the workflow using ESLint + latest SARIF formatter
The link to the failed workflow https://github.com/yongyan-gh/vscode-custom-data/actions/runs/1924302121
and  sarif log file generated: https://github.com/yongyan-gh/vscode-custom-data/suites/5511324619/artifacts/176574998

```
Run github/codeql-action/upload-sarif@v1
  with:
    sarif_file: eslint.sarif
    checkout_path: /home/runner/work/vscode-custom-data/vscode-custom-data
    token: ***
    matrix: null
    wait-for-processing: false
Uploading results
  Processing sarif files: ["eslint.sarif"]
Error details: instance.runs[0].invocations[0].toolConfigurationNotifications[0] additionalProperty "suppressions" exists in instance when not allowed
Error details: instance.runs[0].invocations[0].toolConfigurationNotifications[1] additionalProperty "suppressions" exists in instance when not allowed
Error details: instance.runs[0].invocations[0].toolConfigurationNotifications[[2](https://github.com/yongyan-gh/vscode-custom-data/runs/5397308005?check_suite_focus=true#step:5:2)] additionalProperty "suppressions" exists in instance when not allowed
Error details: instance.runs[0].invocations[0].toolConfigurationNotifications[[3](https://github.com/yongyan-gh/vscode-custom-data/runs/5397308005?check_suite_focus=true#step:5:3)] additionalProperty "suppressions" exists in instance when not allowed
Error details: instance.runs[0].invocations[0].toolConfigurationNotifications[[4](https://github.com/yongyan-gh/vscode-custom-data/runs/5397308005?check_suite_focus=true#step:5:4)] additionalProperty "suppressions" exists in instance when not allowed
Error details: instance.runs[0].invocations[0].toolConfigurationNotifications[[5](https://github.com/yongyan-gh/vscode-custom-data/runs/5397308005?check_suite_focus=true#step:5:5)] additionalProperty "suppressions" exists in instance when not allowed
Error details: instance.runs[0].invocations[0].toolConfigurationNotifications[[6](https://github.com/yongyan-gh/vscode-custom-data/runs/5397308005?check_suite_focus=true#step:5:6)] additionalProperty "suppressions" exists in instance when not allowed
Error: Unable to upload "eslint.sarif" as it is not valid SARIF:
- instance.runs[0].invocations[0].toolConfigurationNotifications[0] additionalProperty "suppressions" exists in instance when not allowed
- instance.runs[0].invocations[0].toolConfigurationNotifications[1] additionalProperty "suppressions" exists in instance when not allowed
- instance.runs[0].invocations[0].toolConfigurationNotifications[2] additionalProperty "suppressions" exists in instance when not allowed
- instance.runs[0].invocations[0].toolConfigurationNotifications[3] additionalProperty "suppressions" exists in instance when not allowed
- instance.runs[0].invocations[0].toolConfigurationNotifications[4] additionalProperty "suppressions" exists in instance when not allowed
- instance.runs[0].invocations[0].toolConfigurationNotifications[5] additionalProperty "suppressions" exists in instance when not allowed
- instance.runs[0].invocations[0].toolConfigurationNotifications[6] additionalProperty "suppressions" exists in instance when not allowed
Error: Unable to upload "eslint.sarif" as it is not valid SARIF:
- instance.runs[0].invocations[0].toolConfigurationNotifications[0] additionalProperty "suppressions" exists in instance when not allowed
- instance.runs[0].invocations[0].toolConfigurationNotifications[1] additionalProperty "suppressions" exists in instance when not allowed
- instance.runs[0].invocations[0].toolConfigurationNotifications[2] additionalProperty "suppressions" exists in instance when not allowed
- instance.runs[0].invocations[0].toolConfigurationNotifications[3] additionalProperty "suppressions" exists in instance when not allowed
- instance.runs[0].invocations[0].toolConfigurationNotifications[4] additionalProperty "suppressions" exists in instance when not allowed
- instance.runs[0].invocations[0].toolConfigurationNotifications[5] additionalProperty "suppressions" exists in instance when not allowed
- instance.runs[0].invocations[0].toolConfigurationNotifications[6] additionalProperty "suppressions" exists in instance when not allowed
```

**Comment by yongyan-gh** [comment]
Looks like the issue is because the `suppressions` property added to invocations.toolConfigurationNotifications is not a valid property

```
      "invocations": [
        {
          "toolConfigurationNotifications": [
            {
              "level": "error",
              "message": {
                "text": "Parsing error: 'import' and 'export' may appear only with 'sourceType: module'"
              },
              "locations": [
                {
                  "physicalLocation": {
                    "artifactLocation": {
                      "uri": "file:///home/runner/work/vscode-custom-data/vscode-custom-data/samples/svg/build/index.ts",
                      "index": 1
                    },
                    "region": {
                      "startLine": 1,
                      "startColumn": 1
                    }
                  }
                }
              ],
              "descriptor": {
                "id": "ESL0999"
              },
              "suppressions": []
            },
```

**Comment by yongyan-gh** [comment]
Have a fix #2464 please review

**Comment by eddynaka** [comment]
Hi @Yiwei-Ding ,

we just released:
https://www.npmjs.com/package/@microsoft/eslint-formatter-sarif

Let us know if you face any problems.
Thank you!
